src/ActionMenu.tsx

```diff
@@ -123,8 +124,13 @@ const Overlay: React.FC<MenuOverlayProps> = ({children, align = 'start', ...over
       onOpen={openWithFocus}
       onClose={onClose}
       align={align}
-      overlayProps={overlayProps}
+      overlayProps={{
+        renderInPortal: false, // render inline for natural focus management
+        sx: merge({zIndex: 1}, propsSx as SxProp), // render above overlay contents (story/internal-components-overlay--memex-nested-overlays)
```

Comment:
this should be > 1 to give room for application folks to put stuff under it 🤔 

will change to 9